shared/services/rocketpool/assets/install/alerting/alertmanager.tmpl

```diff
@@ -40,13 +40,28 @@ receivers:
     discord_configs:
       - webhook_url: "{{ .DiscordWebhookURL.Value }}"
     {{- end }}
-  
+
+    {{- if and .PushoverToken.Value .PushoverUserKey.Value }}
+    pushover_configs:
+      - token: "{{ .PushoverToken.Value }}"
+        user_key: "{{ .PushoverUserKey.Value }}"
+        priority: {{ `'{{ if eq .CommonLabels.severity "critical" }}2{{ else if eq .CommonLabels.severity "warning" }}1{{ else }}0{{ end }}'` }}
```

Comment:
just to open a discussion about the values vis-a-vis alerts.. 

a priority of 1 for warning messages means that the user will get an override of quiet hours and get a notification for warning messages. Now, most alerts (e.g. OSUpdates or RPUpdates) are sending `warning`, with little sending `info`. Perhaps we should review the severities of the rules too in a different PR?